source/_integrations/home_connect.markdown

```diff
@@ -72,7 +76,6 @@ Some devices only have the state `on` and turn off is not supported by the appli
 
 {% important %}
 
-- **Power on** all your appliances during the integration configuration process; otherwise, appliance programs list will be empty.
 - To update the appliance programs list, you can reload the Home Connect integration when an appliance is turned on. If the re-initialization process is not triggered by reload, restart the Home Assistant when an appliance is turned on.
```

Comment:
Is it still needed to reload the config entry to update programs?

Reply:
Yes, it is, the user can also disconnect and connect the appliance to trigger the post-connection update.
Also, I don't know if there's any way to manually trigger the coordinator update.

Reply:
One can call the `homeassistant.update_entity` action targeting one of the integration entities.

Reply:
Should we make a request for the programs automatically when an appliance is turned on and we haven't requested the programs yet?

Reply:
If the use case you are thinking of is the case where the appliance doesn't report any data because it disconnected while it is turned off because , we already fetch all the programs when it is turned on (because the CONNECTED event is received).
I can't think about any other use case right now

Reply:
Great! Thanks for clarifying.

Reply:
This is what we have right now:
https://github.com/home-assistant/home-assistant.io/blob/6b4953ef91a803f5fc9b2b93bdbe337baf82ac93/source/_integrations/home_connect.markdown?plain=1#L51-L53
It doesn't talk about the entities that use the programs specifically, but I think that the statement generalizes enough to include them.

Reply:
Right, that sentence is enough. Then we can just remove the sentence here on line 79.